docs/imviz/import_data.rst

```diff
@@ -97,6 +97,21 @@ is separated from the ``jwst`` pipeline package.
 
 .. _imviz-import-catalogs-api:
 
+Batch Loading Multiple Images
+-----------------------------
+
+To save on performance while loading multiple images into Imviz, you can optionally use
+:meth:`~jdaviz.configs.imviz.helper.Imviz.batch_load` to defer all linking and loading of images
+into the viewer until all data are loaded::
```

Comment:
Good point... _technically_ it delays linking and loading _into the viewer_ until after the data are loaded _into the data collection_, but I think the data collection vs viewer distinction isn't necessary for the user.  How would you feel about the last "loaded" being changed to "parsed" to avoid the confusion without having to explain the dirty details?

Reply:
I suspected that's what you meant, but that's not very clear if you don't know a bunch of subtleties about glue. Maybe going from this
```
...to defer all linking and loading of images into the 
viewer until all data are loaded
```
to
```
...to parse all of the data first with your own Python 
expressions (like a loop, for example), and defer the 
linking and loading of the new images into the viewer 
until after the parsing is complete.
```
It's wordier but clearer?

Reply:
I like that - I made some slight modifications to:

```
to parse all of the data first (within a for loop or multiple calls to ``load_data``, for example), 
and defer the linking and loading of the new data entries into the viewer until after the parsing 
is complete.
```